docs/book/src/reference/metrics.md

```diff
@@ -3,6 +3,53 @@
 By default, controller-runtime builds a global prometheus registry and
 publishes a collection of performance metrics for each controller.
 
+## Exporting Metrics for Prometheus
+
+The project can be enabled to be configured by using `MonitorService` resources from
+the Prometheus Operator. Following the steps to do this setup.
+
+1. Install Prometheus Operator. You can follow up the steps in [Kube Prometheus](https://github.com/coreos/kube-prometheus) project.
+2. Then, enable the project to export the metrics by uncommenting the line `- ../prometheus` in the `config/default/kustomization.yaml` to allow 
+it creates the `ServiceMonitor` resource which is used to do this export. Following an example.
+
+```yaml
+# [PROMETHEUS] To enable prometheus monitor, uncomment all sections with 'PROMETHEUS'.
+- ../prometheus
+```
+
+Note, that now when you install your project in the cluster by the following steps it will create the
+`Service` and `Monitor Service` to export the metrics. 
+
+```yaml
+$ make install
+$ kubectl apply -f config/samples/
```

Comment:
I am not sure if we need this line considering that "metrics" section is not part of "cronjob" tutorial. Personally, I wouldn't list the installation steps at all as by this time, one should already be familiar with how to install the projects (and it will simplify the instructions).